Packs/MongoDBAtlas/Integrations/MongoDBAtlas/MongoDBAtlas.py

```diff
@@ -0,0 +1,586 @@
+        event_type (str): The type of event ('alerts' or 'events').
+    """
+    event['source_log_type'] = event_type
+    add_time_field(event)
+    if event_type == 'alerts':
+        add_entry_status_field(event)
+
+
+################ ALERTS FUNCTIONS ################
+
+
+def remove_alerts_by_ids(alerts: list, ids: list):
+    """
+    Removes alerts from a list based on specified IDs.
+
+    Args:
+        alerts (list): A list of alert dictionaries, each containing an "id" key.
+        ids (list): A list of IDs to be removed from the alerts list.
+
+    Returns:
+        list: A filtered list of alerts excluding any alerts with IDs in the provided ids list.
+    """
+    results = []
+    for alert in alerts:
+        if alert.get('id') not in ids:
+            results.append(alert)
+    return results
+
+
+def get_page_from_last_run_for_alerts(client: Client, page_link: str):
+    """
+    Retrieves alerts based on the last fetched page link or performs an initial fetch.
+
+    Args:
+        client (Client): MongoDB Atlas client.
+        page_link (str): The URL of the last fetched page, if available.
+
+    Returns:
+        dict: The API response containing alerts data.
+    """
+    if page_link:
+        demisto.debug(f'Getting a response from page {page_link}')
+        response = client.get_response_from_page_link(page_link)
+    else:
+        demisto.debug('Initialize the first page')
+        response = client.get_alerts_request(page_num=1, items_per_page=500)
+    return response
+
+
+def create_last_run_dict_for_alerts(links: list, last_page_alerts_ids: list) -> dict:
+    """
+    Updates the last_run dictionary with the current page's self URL and event IDs.
+
+    Args:
+        links (list): Current page links for pagination.
+        last_page_alerts_ids (list): IDs of alerts on the last processed page.
+
+    Returns:
+        dict: Updated last_run dictionary.
+    """
+    return {
+        'page_link': get_page_url(links, page_type='self'),
+        'last_page_alerts_ids': last_page_alerts_ids
+    }
+
+
+def fetch_alert_type(client: Client, fetch_limit: int, last_run: dict):
+    """
+    Fetches alerts until fetch_limit is reached, or no more alerts are available.
+
+    Args:
+        client (Client): MongoDBAtlas client.
+        fetch_limit: The maximum number of alerts to fetch.
+        last_run (dict): Dictionary containing data from the previous run.
+
+    Returns:
+        - The list containing all fetched events.
+        - The object to save for the next run.
+    """
+
+    demisto.debug('Start to fetch alerts')
+    page_link = last_run.get('page_link', "")
+
+    response = get_page_from_last_run_for_alerts(client, page_link)  # get the last page or the first page
+
+    links = response.get('links')
+    results = response.get('results')
+
+    last_page_alerts_ids = last_run.get('last_page_alerts_ids', [])
+
+    events = remove_alerts_by_ids(results, last_page_alerts_ids)
```

Comment:
If we're talking about alerts here, let's keep it that way so it won't be confusing.